test/client/fixtures/runner/api/press-test.js

```diff
@@ -428,4 +428,13 @@ $(document).ready(function () {
 
         document.body.appendChild(iframe);
     });
+
+    asyncTest('Press action should accept only string as argument', function () {
+        actionsAPI.press(100);
+        setTimeout(function () {
+            equal(currentErrorType, ERROR_TYPE.incorrectPressActionArgument);
+
+            start();
+        }, 300);
+    });
```

Comment:
I think it's better to place this test into [the `parse keys string` module](https://github.com/georgiy-abbasov/testcafe-phoenix/blob/f11cb0ac2fd6b087ad19489d99ce8ce9babe7893/test/client/fixtures/runner/api/press-test.js#L122) after the `press incorrect keys combination` test